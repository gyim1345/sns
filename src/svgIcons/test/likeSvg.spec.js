import React from 'react';
import { mount } from 'enzyme';
import LikeSvg from '../LikeSvg';

describe('<LikeSvg />', () => {
  it('renders <svg>', () => {
    const wrapper = mount(<LikeSvg />);

    expect(wrapper.html()).toMatch('<svg');
    expect(wrapper).toMatchSnapshot();
  });
})

