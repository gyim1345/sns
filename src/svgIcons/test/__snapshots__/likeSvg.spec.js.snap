// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LikeSvg /> renders <svg> 1`] = `
<LikeSvg>
  <svg
    heigth="24"
    id="increaseLikeSvg"
    style={
      Object {
        "marginBottom": "5px",
        "marginLeft": "16px",
        "marginRight": "15px",
      }
    }
    type="button"
    viewBox="0 0 50 50"
    width="24"
  >
    <path
      className="heart"
      d="M 34.3 3.5 C 27.2 3.5 24 8.8 24 8.8 s -3.2 -5.3 -10.3 -5.3 C 6.4 3.5 0.5 9.9 0.5 17.8 s 6.1 12.4 12.2 17.8 c 9.2 8.2 9.8 8.9 11.3 8.9 s 2.1 -0.7 11.3 -8.9 c 6.2 -5.5 12.2 -10 12.2 -17.8 c 0 -7.9 -5.9 -14.3 -13.2 -14.3 Z m -1 29.8 c -5.4 4.8 -8.3 7.5 -9.3 8.1 c -1 -0.7 -4.6 -3.9 -9.3 -8.1 c -5.5 -4.9 -11.2 -9 -11.2 -15.6 c 0 -6.2 4.6 -11.3 10.2 -11.3 c 4.1 0 6.3 2 7.9 4.2 c 3.6 5.1 1.2 5.1 4.8 0 c 1.6 -2.2 3.8 -4.2 7.9 -4.2 c 5.6 0 10.2 5.1 10.2 11.3 c 0 6.7 -5.7 10.8 -11.2 15.6 Z"
    />
  </svg>
</LikeSvg>
`;
